openesdh-repo/src/main/java/dk/openesdh/repo/audit/CaseWorkflowReviewOutcomeExtractor.java

```diff
@@ -0,0 +1,30 @@
+package dk.openesdh.repo.audit;
+
+import java.io.Serializable;
+import java.util.Map;
+
+import org.alfresco.repo.workflow.WorkflowModel;
+import org.alfresco.service.cmr.workflow.WorkflowService;
+import org.alfresco.service.namespace.QName;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+@Service("audit.dk.openesdh.CaseWorkflowReviewOutcomeExtractor")
+public class CaseWorkflowReviewOutcomeExtractor extends AbstractAnnotatedDataExtractor {
+
+    @Autowired
+    @Qualifier("WorkflowService")
+    private WorkflowService workflowService;
+
+    @Override
+    public Serializable extractData(Serializable value) throws Throwable {
+        Map<QName, Serializable> taskProps = workflowService.getTaskById(value.toString()).getProperties();
+        return taskProps.entrySet().stream()
```

Comment:
why not to get this single property directly?
taskProps.get(WorkflowModel.PROP_OUTCOME_PROPERTY_NAME)